scripts/release/issue-list.sh

```diff
@@ -17,9 +17,10 @@
 programname=$0
 function usage()
 {
-    echo "usage: $programname -r repository -m milestone"
+    echo "usage: $programname -r repository -m milestone -f format"
     echo "  -r  repository name"
     echo "  -m  milestone id"
+    echo "  -f  output format ('human' or 'jsonstring')"
```

Comment:
I don't think so. right now you are spreading the conversion of this output over three places. Here you are dealing with new lines, somewhere else you are doing some URL encoding (which I believe is not necessary if the payload is JSON) and last but not least you are calling a generate_json function. IMO, this script does not have to have a format flag and I think you don't need to url encode, just call issue-list and create your JSON payload. We already have jq installed, it is the swiss army knife of JSON manipulation. What you are after should be as simple as : 

    ./scripts/release/issue-list.sh -r minishift -m 17   | jq -Rn 'inputs|tojson' | jq -s '{body:  add } '

Convert string to JSON using _tojson_ and then wrap this into the payload for the GitHub API call. 

If you want to pack this into a function, you can pass the default output of the script to your _create_json_ function and in there you do the jq conversion and returns JSON payload for the API call.